gigareview/console/tests/acceptance/EMReportJob.feature

```diff
@@ -0,0 +1,31 @@
+Feature:
+  As an editor
+  I want the automatically downloaded EM reports spreadsheet to be ingested automatically
+  So that I can maintain an up-to-date database of peer reviews to be curated and to be shown on GigaReviews
+
+  Background:
+    Given the database is reset
+
+  @ok
+  Scenario: EM manuscript report is downloaded and saved to manuscript table
+    Given the file is on the sftp server
+    And the file ingester has run
+    And the EM "manuscripts" report spreadsheet is downloaded
+    When the queue job is pushed to "manuscripts" worker
```

Comment:
Jobs are pushed into a queue but they are not pushed into a worker?  The worker executes the jobs in a queue so you could say in this statement, `When the "manuscripts" worker executes the queue jobs`.